math/moserdebruijnsequence/sequence.go

```diff
@@ -0,0 +1,37 @@
+// The Moser-de Bruijn sequence is the sequence obtained by
+// adding up the distinct powers of the number 4 (For example 1, 4, 16, 64, etc).
+
+package main
+
+import "fmt"
+
+func main() {
```

Comment:
Also, you should remove the main function.
You can change your `getSequenceNumber` function to `MoserDeBruijnSequence` instead and return the final result.